geoapps/inversion/constants.py

```diff
@@ -504,6 +504,15 @@
         "value": True,
         "verbose": 3,
     },
+    "out_group": {
+        "label": "SimPEG group",
+        "value": None,
+        "groupType": "{55ed3daf-c192-4d4b-a439-60fa987fe2b8}",
+        "group": "Drag-and-drop options",
```

Comment:
does this need a group name?  There won't be anything in the group with it, will there?

Reply:
That's Kris' call. He likes the group better as the optional goes on it.